ui/app/templates/components/text-file.hbs

```diff
@@ -60,7 +60,7 @@
   {{else}}
     <div class="control is-expanded">
       <div class="file has-name is-fullwidth">
-        <label class="file-label">
+        <div class="file-label">
           <input class="file-input" type="file" {{on "change" this.pickedFile}} data-test-text-file-input={{true}} />
```

Comment:
missing label here too. Perhaps we update the span on lin 65 to `<label`?

Reply:
Since the aria-label you propose is the same as the text on line 67 I think we should just use that, otherwise screen readers will get duplicate readouts. Also, I just noticed the button on line 78 has only an icon not a label, so we should fix that too 😬